### DISCUSSION TITLE
🪄 Element Plus Nightly Released

### DISCUSSION THREAD

**Discussion by sxzz** [element-plus/element-plus#5056] [Show and tell]
The package name of nightly version is `@element-plus/nightly`. The naming of version is `0.0.yyyymmdd` (such as `0.0.20211230`).

包名为 `@element-plus/nightly`。命名规则为 `0.0.年月日`（比如 `0.0.20211230`）。

It will publish every night (GMT+8 00:00). If you encounter bugs that have been fixed but not released, you can try this package.

北京时间每天 0 时发布。如果你遇到已经修复了的 bug，但是还没发版，可以尝试使用这个包。

Try:
```jsonc
// package.json
{
  "dependencies": {
    "element-plus": "npm:@element-plus/nightly@latest"
  }
}
```

> **Warning** **Use at your own risk! 使用风险自负！**

**Comment by lixiaofa** [comment]
@sxzz 
发布@element-plus 到npm 
npm 登录放在了哪里  ， 是需要我自己加吗
